Snakefile

```diff
@@ -442,6 +489,12 @@ rule download_amazon_acl10:
     shell:
         "curl -Lf 'http://www.uni-weimar.de/medien/webis/corpora/corpus-webis-cls-10/cls-acl10-unprocessed.tar.gz' -o {output}"
 
+rule download_paracrawl:
+    output:
+        "data/downloaded/paracrawl/{lang1}_{lang2}.tmx.gz"
+    shell:
+        "curl -lf 'https://s3.amazonaws.com/web-language-models/paracrawl/release1.2/paracrawl-release1.2.{wildcards.lang1}-{wildcards.lang2}.withstats.filtered-bicleaner.tmx.gz' -o {output}"
```

Comment:
What's the advantage of using `curl` vs `wget` here?

Reply:
For other commands, I found `curl` preferable so I can pipe its output to things.

There is a slight advantage to `wget` for thing that aren't being piped, which is that the download can be resumed if it's interrupted. But it doesn't seem worth it to add a dependency on wget when most of the rules are using curl.